llvm/lib/Analysis/ModuleSummaryAnalysis.cpp

```diff
@@ -787,7 +804,9 @@ static void computeVariableSummary(ModuleSummaryIndex &Index,
                                    SmallVectorImpl<MDNode *> &Types) {
   SetVector<ValueInfo, std::vector<ValueInfo>> RefEdges;
   SmallPtrSet<const User *, 8> Visited;
-  bool HasBlockAddress = findRefEdges(Index, &V, RefEdges, Visited);
+  bool RefIFunc = false;
```

Comment:
Suggest "RefLocalIFunc"

Reply:
Also, can you add this case to your new test? I think it is only testing the call case.

Reply:
Renamed and added `@var = constant { [1 x ptr] } { [1 x ptr] [ptr @callee]}` in the test.